admin/lib/generators/solidus_admin/install/install_generator.rb

```diff
@@ -3,7 +3,9 @@
 module SolidusAdmin
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      source_root File.expand_path('templates', __dir__)
+      class_option :lookbook, type: :boolean, default: !!ENV['SOLIDUS_ADMIN_LOOKBOOK'], banner: 'Install Lookbook for component previews'
```

Comment:
I tried to run ` bin/rails g solidus_admin:install --help`

```
Running Rails from: /Users/kennyadsl/Code/nebulab/solidus/sandbox
Usage:
  rails generate solidus_admin:install [options]

Options:
  [--skip-namespace], [--no-skip-namespace]                              # Skip namespace (affects only isolated engines)
  [--skip-collision-check], [--no-skip-collision-check]                  # Skip collision check
  [--lookbook=Install Lookbook for component previews], [--no-lookbook]  # Indicates when to generate lookbook

Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist

Description:
    Create solidus admin files for install generator.
```

And I think there's something off with the `lookbook` option. It's setting the banner in the value of the option, which doesn't seem correct to me.

Reply:
@kennyadsl good catch, I copied it from the core install generator, but maybe being a different superclass the syntax is different, fixing it 👍